src/include/crowbar/complex.rb

```diff
@@ -730,7 +730,7 @@ def show_custom_repos_widget
     def InitRPRepos(id)
       # find the initial location now
       if @repos_location.empty?
-        @repos_location = "custom"
+        @repos_location = "local"
```

Comment:
Actually  `@repos_file = "/opt/dell/crowbar_framework/config/repos-cloud.yml"`, but I still dont know where that is.